src/filters/searchFilter.js

```diff
@@ -0,0 +1,19 @@
+const elasticlunr = require("elasticlunr");
```

Comment:
What does this library do?

Reply:
this is for searching in our blog posts.

Reply:
This goes against the quality parameters that we have established. The last commit for this library was 5 years ago and the project is abandoned.

We have talked about library selection criterias numerous times over the past few months.

Reply:
let me resolve this!